scores/2023-12-28-animal-crossing-new-leaf.md

```diff
@@ -0,0 +1,5 @@
+---
+title: HippoPenny Score
+layout: scoredetail
```

Comment:
can you please share a screenshot somewhere?

Reply:
done